src/site/content/en/blog/variable-fonts/index.md

```diff
@@ -7,358 +7,694 @@ description: >
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="roboto-dance.mp4" type="video/mp4">
+  </video>
+  <figcaption>
+    Roboto Flex in random combinations of Width and Weight
   </figcaption>
 </figure>
 
-The [OpenType specification](https://docs.microsoft.com/en-gb/typography/opentype/spec/dvaraxisreg)
-specifies other axis, such as **Width**, **Optical Size**, **Italic** and
-**Slant**. All of these axes share the same default master, and we can combine
-them to explore an exponential number of typographic styles, like powers of 2,
-3, and 4 do to numbers.
-
-Amstelvar also has three masters in a Width axis: Again the Regular is at the
-center of the axis, and two masters, narrow and wider, are at the opposite ends
-of this axis. These not only provide all the widths of the Regular style, but
-also all the widths and weights combined.
-
-Among Amstelvar's registered axes (Width, Weight and Optical Size) there are
-thousands of styles. This may seem like massive overkill, but consider that
-Amstelvar only supports the Latin writing system. Considering the needs of all
-the world's scripts, and many of today's typography applications, the quality
-of the reading experience can be remarkably enhanced by this diversity of type
-styles within a font. And, if it is without performance penalty, the user can
-use a few or as many styles as they wish–it's up to their design.
-
-### Italics are slightly different
-
-The way that italics are handled in variable fonts is interesting, as there are
-two different approaches. Typefaces like Helvetica or Roboto have 
-interpolation-compatible contours, so their Roman and Italic styles can be interpolated
+This means there are thousands of styles! This may seem like massive overkill,
+but the quality of the reading experience can be remarkably enhanced by this
+diversity of type styles. And, if it is without performance penalty, web
+developers can use a few or as many styles as they wish–it's up to their design.
+
+### Italics
+
+The way that Italics are handled in variable fonts is interesting, as there are
+two difference approaches. Typefaces like Helvetica or Roboto have interpolation
+compatible contours, so their Roman and Italic styles can be interpolated
 between and the **Slant** axis can be used to get from Roman to Italic.
 
 Other typefaces (such as Garamond, Baskerville, or Bodoni) have Roman and Italic
-glyph contours that are not interpolation-compatible. For example, the contours
+glyph contours that are not interpolation compatible. For example, the contours
 that typically define a Roman lowercase "n" do not match the contours used to
 define an Italic lowercase "n". Instead of interpolating one contour to the
 other, the **Italic** axis toggles from Roman to Italic contours.
 
 <figure class="w-figure">
   <img src="Amstelvar-Ital-axis.png"
   alt="Example of the Weight Axes for the typeface Amstelvar">
-  <figcaption class="w-figcaption">
+  <figcaption>
```

Comment:
```suggestion
  <figcaption class="w-figcaption">
```
Need to keep our styling consistent, file a feature request with our engineering team if you want to change how captions are presented